hack/install_golangci.sh

```diff
@@ -1,5 +1,17 @@
-#!/bin/bash -e
+#!/bin/bash
+
```

Comment:
We want `set -eo pipefail` here.

Reply:
If we do this, we'll fail in the version check below when golangci-lint is not installed. I tried that but it didn't work.

Reply:
I don't have a strong opinion on setting `pipefail`, but if it's really desired, then one solution could be...